relay/polkadot/src/lib.rs

```diff
@@ -2977,200 +2979,6 @@ sp_api::impl_runtime_apis! {
 			Vec<frame_benchmarking::BenchmarkBatch>,
 			sp_runtime::RuntimeString,
 		> {
-			use frame_support::traits::WhitelistedStorageKeys;
```

Comment:
To make it compile?

Reply:
This was duplicated when we merged `main`, it is moved above.